core/scss/utilities/mixins/grid/_centered-column.scss

```diff
@@ -1,20 +1,26 @@
 @charset "UTF-8";
 
 //
-// @centered-column($grid: $grid-media, $screens: $grid-screens)
+// @centered-column($grid: $grid-media, $margin: $screen-margins)
 //
 // Centers a single cellin a container based on max width in breakpoints.
 //
 // Style guide: Mixins.Grid.centered-column
 //
-@mixin centered-column($grid: $grid-media, $screens: $grid-screens, $gutters: $grid-gutters) {
+@mixin centered-column($grid: $grid-media, $screens: $grid-screens, $margin: $screen-margins) {
   @include margin(0 auto);
   @each $code, $breakpoint in $grid {
-    $max-width: map-get($screens, $code);
-    $gutter: map-get($gutters, $code);
+    $spacing: map-get($margin, $code);
 
     @media #{$breakpoint} {
-      max-width: $max-width - $gutter;
+      @if breakpoint-max($code, $grid) == null {
+        $max-width: map-get($screens, $code);
+        max-width: $max-width;
+        width: calc(100% - ($spacing * 2));
```

Comment:
I think you meant
#{$spacing * 2} instead of ($spacing * 2) here - However, I think it actually works better if this calculation isn't used for the 2xl breakpoint. 

Copying my comment below here: "Do we need screen margins for 2xl? I kind of like it better before when it's just using 100px from the xl breakpoint until it reaches 1500px."

Reason being:
At 1499px (largest end of XL bp) - the centered container width is 1499 - 100 x 2  = 1299px
At 1500px (start of 2XL) - the centered container width is 1500 - 150 x 2 = 1200px,  which means when it transitions from XL to 2XL, the centered container actually shrinks by 99px. So it seems better if XL and 2XL have the same screen margin value.

All the other changes look great - totally agree with those.